src/main/java/io/cryostat/recordings/RecordingTargetHelper.java

```diff
@@ -428,18 +447,17 @@ private Future<Void> deleteRecording(
                                     recordingMetadataManager.deleteRecordingMetadataIfExists(
                                             connectionDescriptor.getTargetId(), recordingName);
                                     if (issueNotification) {
-                                        notificationFactory
-                                                .createBuilder()
-                                                .metaCategory(DELETION_NOTIFICATION_CATEGORY)
-                                                .metaType(HttpMimeType.JSON)
-                                                .message(
-                                                        Map.of(
-                                                                "recording",
-                                                                linkedDesc,
-                                                                "target",
-                                                                connectionDescriptor.getTargetId()))
-                                                .build()
-                                                .send();
+                                        if (isSnapshot) {
+                                            this.issueNotification(
+                                                    connectionDescriptor.getTargetId(),
+                                                    linkedDesc,
+                                                    SNAPSHOT_DELETION_NOTIFICATION_CATEGORY);
+                                        } else {
+                                            this.issueNotification(
+                                                    connectionDescriptor.getTargetId(),
+                                                    linkedDesc,
+                                                    DELETION_NOTIFICATION_CATEGORY);
```

Comment:
I think it would be a little cleaner to just set the value of a `String notificationCategory` local variable based on `isSnapshot`, and then call `this.issueNotification(connectionDescriptor.getTargetId(), linkedDesc, notificationCategory)` identically either way.